target/linux/mediatek/dts/mt7981b-asus-rt-ax52.dts

```diff
@@ -0,0 +1,290 @@
+/dts-v1/;
```

Comment:
You didn't include SPDX line at the top of this file. It makes DTS fall under OpenWrt's default licensing which is GPL 2.0.

DTS files for mt7981* are licensed under GPL 2.0 and MIT. Please license your file the same way to make them compatible. For that you need
```
// SPDX-License-Identifier: GPL-2.0-only OR MIT
```